src/pages/EditStudent.tsx

```diff
@@ -35,27 +35,27 @@ const EditStudent = () => {
   }
   const [stage, setStage] = useState(STAGES.NAME);
   const [studentName, setStudentName] = useState(
-    isEdit ? api.currentStudent?.name : ""
+    isEdit ? Util.getCurrentStudent()?.name : ""
```

Comment:
don't call **Util.getCurrentStudent()** Multiple times in same page store it variable and reuse it

Reply:
follow same thing in all pages